src/delta.rs

```diff
@@ -171,10 +170,11 @@ fn handle_hunk_meta_line(
         cli::SectionStyle::Plain => panic!(),
     };
     let ansi_style = Blue.normal();
-    let (code_fragment, line_number) = parse::parse_hunk_metadata(&line);
+    let (raw_code_fragment, line_number) = parse::parse_hunk_metadata(&line);
+    let code_fragment = prepare(raw_code_fragment, config.tab_width, false);
```

Comment:
Previously we were only calling the `prepare` function on code lines within diff hunks; this change calls `prepare` on the code fragment line captured by git to act as a hunk context marker.